services/permits/app/compare_extraction_results.py

```diff
@@ -103,6 +117,8 @@ def validate_condition(csv_pairs):
         auto_extracted_content = create_content_instances(auto_extracted_df)
         manual_extracted_content = create_content_instances(manual_extracted_df)
 
+        print(f'Found {len(auto_extracted_content)} conditions in {auto_extracted_csv} and {len(manual_extracted_content)} conditions in {manual_extracted_csv}')
```

Comment:
Was this print statement meant to be added as well?

Reply:
Indeed it was intentional